vimrc

```diff
@@ -165,9 +165,10 @@ map <Leader>i mmgg=G`m<CR>
 syntax on
 set t_Co=256
 set background=dark
-colorscheme solarized
+colorscheme smyck
```

Comment:
Great question.

If everyone loves Solarized, then I'm happy to move it to my .vimrc.local.

If we're on Solarized b/c that came with the thoughtbot dotfiles and we haven't considered it further, might I suggest we move to Smyck? I find the color palette and contrast levels much nicer. Solarized has some really dark blues that are nearly indistinguishable from a dark terminal window.

To see what Smyck looks like, take a look at the screenshots on this page: http://color.smyck.org/